native/packages/react-native-bpk-component-progress/readme.md

```diff
@@ -0,0 +1,48 @@
+# react-native-bpk-component-progress
+
+> Backpack React Native Progress component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-progress --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import BpkProgress from 'react-native-bpk-component-progress';
+import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  },
+});
+
+export default class App extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <BkpProgress min={0} max={100} value={10} />
+        <BkpProgress min={0} max={100} value={10} type="Bar" />
+      </View>
+    );
+  }
+}
+```
+
+## Props
+
+| Property   | PropType                | Required | Default Value |
+| ---------- | ----------------------- | -------- | ------------- |
+| min        | number                  | false    | 0             |
+| max        | number                  | false    | 100           |
+| value      | number                  | true     | -             |
+| type       | oneOf('Default', 'Bar') | false    | Default       |
```

Comment:
We don't typically have props like this in UpperCamelCase I believe

Reply:
you mean it should be `default` and `bar`?

Reply:
Exactly 👌